src/components/atoms/button/stories/button.stories.native.js

```diff
@@ -5,18 +5,17 @@ import { Text, View } from 'react-native';
 import { SafeView } from 'storybook/story-components-native';
 
 import Button from '../';
+import { ThemeProvider } from 'store';
 
 storiesOf('Atoms/Button', module)
   .addDecorator(storyFn => (
-    <SafeView>
-      {storyFn()}
-      <View>
-        <Text style={{ textAlign: 'center' }}>Native Button</Text>
-      </View>
-    </SafeView>
+    <ThemeProvider>
```

Comment:
_Should_ be able to add a global decorator to handle the theming 👍 
That way, this block won't need to be repeated 👍